.travis.yml

```diff
@@ -36,7 +57,7 @@ script:
 
 after_success:
   - |
-    if [[ "${TRAVIS_BRANCH}" == master && "${TRAVIS_PULL_REQUEST}" == false && $TRAVIS_PYTHON_VERSION == 3.7 ]];
+    if [[ "${TRAVIS_BRANCH}" == master && "${TRAVIS_PULL_REQUEST}" == false && $(echo "$TRAVIS_PYTHON_VERSION >= 3.6" | bc -l) == 1 ]];
```

Comment:
I think we want to keep this a check specifically for 3.7, otherwise once we have 3.6 and 3.8 (for example) each CI run will try and push the docs. It isn't that we need 3.7 specifically, just that we only want one version to push.